webapp/components/mainnetLiveModal/index.tsx

```diff
@@ -57,7 +60,10 @@ const Content = function ({ img: Img }: { img: ComponentType }) {
         </ButtonLink>
         <Button
           height="h-5"
-          onClick={() => setHideMainnetLiveModal(true)}
+          onClick={function () {
+            setHideMainnetLiveModal(true)
+            setNetworkType('mainnet')
```

Comment:
Forcing the network here makes sense IMO. @gndelia do you think this is ok or may be redundant/not needed?

Reply:
I think this is correct. The modal appears in the app, regardless of which chain the user is. If the user was in Testnet, and we did not do this, the user would click "Start using mainnet" and would continue on Testnet...